.github/bin/concepts/update.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+
+# Update the concepts summary
+dotnet run -p .github/bin/concepts
+
+# Format the documents
+npx prettier@2.0.4 --write reference/README.md reference/references.json
+
+# Add the updated concept summary files
+git add reference/README.md
+git add reference/references.json
+
+# Check if there is nothing to commit (i.e. no changes to the concepts summary)
+if [ -z "$(git status --porcelain)" ]; then
+    echo "No changes to the concepts summary"
+    exit 0
+fi
+
+# Checkout to new branch
+BRANCH="conceptssummary-$(date +%Y%m%d%H%M%S)"
```

Comment:
What do you think about renaming these to `summaries/concept/$date` or similar? That way they don't interfere as much with tab completion.

Reply:
I like it. I'll update in a follow-up PR.